pegasus/sites.v3/code.org/views/featured_tutorial.html

```diff
@@ -0,0 +1,15 @@
+<div class="col-25 col-rtl-right" style="padding: 10px">
+  <a href="%url%" style="font-family: 'Gotham 4r', sans-serif; color: black; text-decoration: none">
+    <div style="border: 2px solid rgb(187, 187, 187); border-radius: 10px; height: 270px; overflow: hidden; text-align: center">
+      <div style="width: 100%; height: 120px; overflow: hidden">
+        <img src="%image%" style="width: 100%"/>
+      </div>
+      <div style="font-family: 'Gotham 5r', sans-serif; color: #5b6770; font-size: 24px; line-height: 1.5; padding: 5px 5px 0px 5px">
+        %title%
+      </div>
+      <div style="font-size: 13px; line-height: 1.5; color: #5b6770; padding: 5px 5px 0px 5px">
```

Comment:
Curious about the choice to use inline styles here. It seems like we often try to separate these rules out.